code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -204,7 +204,8 @@
 	if(M.a_intent == INTENT_DISARM)
 		if(M.buckled_mobs && (src in M.buckled_mobs) && M.riding_datum)
 			M.riding_datum.force_dismount(src)
-	dna.species.spec_attack_hand(M, src)
+	if(istype(M))

```

Comment:
I've fixed the problem, but in a different way.  Attack hand should NEVER have had a human as an argument, it should have just been a mob, like the rest of the attack_hand procs.  As such, it's been re-written to the point that this is no longer a concern.